src/app/zap-templates/zcl/data-model/chip/web-rtc-provider-cluster.xml

```diff
@@ -0,0 +1,145 @@
+  <domain name="General"/>
+
+  <enum name="StreamTypeEnum" type="enum8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item name="Internal" value="0x00"/>
+    <item name="Recording" value="0x01"/>
+    <item name="Analysis" value="0x02"/>
+    <item name="LiveView" value="0x03"/>
+  </enum>
+
+  <bitmap name="WebRTCMetadataOptions" type="bitmap8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <field name="DataTLV" mask="0x01"/>
+  </bitmap>
+
+  <enum name="WebRTCEndReasonEnum" type="enum8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item name="ICE_FAILED" value="0x01"/>
+    <item name="ICE_TIMEOUT" value="0x02"/>
+    <item name="USER_HANGUP" value="0x03"/>
+    <item name="USER_BUSY" value="0x04"/>
+    <item name="REPLACED" value="0x05"/>
+    <item name="NO_USER_MEDIA" value="0x06"/>
+    <item name="INVITE_TIMEOUT" value="0x07"/>
+    <item name="ANSWERED_ELSEWHERE" value="0x08"/>
+    <item name="OUT_OF_RESOURCES" value="0x09"/>
+    <item name="MEDIA_TIMEOUT" value="0x0A"/>
+    <item name="LOW_POWER." value="0x0B"/>
+    <item name="UNKNOWN_ERROR" value="0x0C"/>
+  </enum>
+
+  <struct name="ICEServerStruct" apiMaturity="provisional">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item fieldId="1" name="urls" array="true" type="char_string"/>
+    <item fieldId="2" name="username" type="char_string" optional="true"/>
+    <item fieldId="3" name="credential" type="char_string" optional="true"/>
+    <item fieldId="4" name="CAID" type="int16u" optional="true"/>
+  </struct>
+
+  <struct name="WebRTCSessionStruct" apiMaturity="provisional">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item fieldId="1" name="ID" type="int16u"/>
+    <item fieldId="2" name="PeerNodeId" type="node_id"/>
+    <item fieldId="3" name="PeerFabricIndex" type="fabric_idx"/>
+    <item fieldId="4" name="StreamType" type="StreamTypeEnum" min="0x00" max="0x03"/>
+    <item fieldId="5" name="VideoStreamID" type="int16u" isNullable="true"/>
+    <item fieldId="6" name="AudioStreamID" type="int16u" isNullable="true"/>
+    <item fieldId="7" name="MetadataOptions" type="WebRTCMetadataOptions" default="0x00" min="0x00" max="0x01"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <domain name="General"/>
+    <name>WebRTC Transport Provider</name>
+    <code>0x0553</code>
+    <define>WEB_RTC_TRANSPORT_PROVIDER_CLUSTER</define>
+    <description>The WebRTC transport provider cluster provides a way for stream providers (e.g. Cameras) to stream or receive their data through WebRTC.</description>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="CURRENT_SESSIONS" type="array" entryType="WebRTCSessionStruct" default="0">CurrentSessions</attribute>
```

Comment:
What does `default="0"` even mean for an array?  I know the spec says it, but the spec is nonsense there.

Reply:
Fixed in spec & pulled here via alchemy